libgadget/blackhole.c

```diff
@@ -611,6 +797,26 @@ blackhole_accretion_ngbiter(TreeWalkQueryBHAccretion * I,
         } while(!__atomic_compare_exchange_n(&(BHP(other).SwallowID), &readid, newswallowid, 0, __ATOMIC_RELAXED, __ATOMIC_RELAXED));
     }
 
+    /*************************************************************************/
+
+    /* Collect surrounding DM/Star (optional gas) density/velocity for DF computation */
+    if(P[other].Type == 1 || P[other].Type == 4 || (P[other].Type == 0 && All.BH_DynamicalFriction == 0) ){
```

Comment:
At this point I realised that BH_DynamicalFriction == 0 means Dynamical Friction is ON and BH_DynamicalFriction == 3 means it is OFF! That's odd.

Reply:
In any case, BH_DynamicalFrictionMethod or Mode / Type is more specific than DynamiccalFriction alone.